commit e66bb8c9fb0678cb1086320f931d805491b747f7
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:49 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index 562806b..fdd11d3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -32,7 +32,7 @@ AC_PROG_CC
 AC_PROG_INSTALL
 
 # Checks for pkg-config packages
-XAW_CHECK_XPRINT_SUPPORT(XCLIPBOARD)
+PKG_CHECK_MODULES(XCLIPBOARD, xaw7)
 AC_SUBST(XCLIPBOARD_CFLAGS)
 AC_SUBST(XCLIPBOARD_LIBS)
 
